test/specs/authentication.e2e.ts

```diff
@@ -31,7 +31,7 @@ describe('Authentication', async () => {
     await utilities.installExtensions();
     await utilities.reloadAndEnableExtensions();
     await testSetup.setUpTestingEnvironment();
-    await testSetup.createProject('Standard');
+    await testSetup.createInitialProject('Standard');
     await utilities.reloadAndEnableExtensions();
     await utilities.verifyAllExtensionsAreRunning();
```

Comment:
Function name is now changed to `verifyExtensionsAreRunning()`.

```suggestion
    await utilities.verifyExtensionsAreRunning();
```